doc/lms.pgsql

```diff
@@ -1261,7 +1261,8 @@ CREATE TABLE networks (
 	dhcpend varchar(16) 	DEFAULT '' NOT NULL,
 	disabled smallint 	DEFAULT 0 NOT NULL,
 	notes text		DEFAULT '' NOT NULL,
-	vlanid smallint DEFAULT NULL,
+	vlanid smallint DEFAULT NULL
+	    CONSTRAINT networks_vlanid_fkey REFERENCES vlans (id) ON DELETE SET NULL ON UPDATE CASCADE,
 	hostid integer NULL
```

Comment:
Kolejność tabel może zrobić problem - odwołujesz się do tabeli **vlans**, który jeszcze na tym etapie nie istnieje.

Reply:
Travis nic nie zgłaszał.
Ok zrobię test u siebie na środowisku od 0.

Reply:
Można to oczywiście obejść robią wszystkie FOREIGN KEYS INITIALLY DEFERRED, ale nie każdy o tym wie.

Reply:
Mimo to przeniosłem tworzenie tabeli vlans nad tabelę networks.